facebook/delphiFacebook/R/contingency_variables.R

```diff
@@ -329,82 +332,836 @@ remap_responses <- function(df) {
+  # # (respondents to V3 OR (yes to V1))
+  if (all(c("b_vaccinated", "b_accept_vaccine") %in% names(df))) {
+    df$b_vaccinated_or_accept <- as.numeric(
+      (!is.na(df$b_vaccinated) & df$b_vaccinated == 1) | df$b_accept_vaccine == 1
     )
   } else {
-    df$b_65_or_older <- NA_real_
+    df$b_vaccinated_or_accept <- NA_real_
   }
 
+  # accept_vaccine_defyes
+  # accept_vaccine_probyes
+  # accept_vaccine_probno
+  # accept_vaccine_defno
+  # # (option chosen) / # V3 responses
   if ("mc_accept_cov_vaccine" %in% names(df)) {
-    df$b_hesitant_cov_vaccine <- as.numeric(
-      df$mc_accept_cov_vaccine == "prob not vaccinate" | df$mc_accept_cov_vaccine == "def not vaccinate"
+    df$b_accept_vaccine_defyes <- as.numeric(df$mc_accept_cov_vaccine == "def vaccinate")
+    df$b_accept_vaccine_probyes <- as.numeric(df$mc_accept_cov_vaccine == "prob vaccinate")
+    df$b_accept_vaccine_probno <- as.numeric(df$mc_accept_cov_vaccine == "prob not vaccinate")
+    df$b_accept_vaccine_defno <- as.numeric(df$mc_accept_cov_vaccine == "def not vaccinate")
+  } else {
+    df$b_accept_vaccine_defyes <- NA_real_
+    df$b_accept_vaccine_probyes <- NA_real_
+    df$b_accept_vaccine_probno <- NA_real_
+    df$b_accept_vaccine_defno <- NA_real_
+  }	
+
+  # vaccine_likely_friends
+  # Percentage more likely to get vaccinated if recommended by friends & family
+  # # more likely / #V4 responses
+  # made elsewhere and renamed in `rename_responses`
+
+  # vaccine_likely_local_health
+  # Percentage more likely to get vaccinated if recommended by local healthcare
+  # workers
+  # # more likely / #V4 responses
+  # made elsewhere and renamed in `rename_responses`
+
+  # vaccine_likely_who
+  # Percentage more likely to get vaccinated if recommended by WHO
+  # # more likely / # V4 responses
+  # made elsewhere and renamed in `rename_responses`
+
+  # vaccine_likely_govt_health
+  # Percentage more likely to get vaccinated if recommended by government 
+  # health officials
+  # # more likely / # V4 responses
+  # made elsewhere and renamed in `rename_responses`
+
+  # vaccine_likely_politicians
+  # Percentage more likely to get vaccinated if recommended by politicians
+  # # more likely / # V4 responses
+  # made elsewhere and renamed in `rename_responses`
+
+  # vaccine_likely_doctors
+  # Percentage more likely to get vaccinated if recommended by doctors and 
+  # other health professionals
+  # # more likely / # V4 responses
+  # made elsewhere and renamed in `rename_responses`
+
+  # worried_vaccine_sideeffects
+  # Percentage very or moderately concerned about side effects
+  # #(very concerned OR moderately concerned) / # V9 responses
+  if ("mc_concerned_sideeffects" %in% names(df)) {	
+    df$b_worried_vaccine_sideeffects <- as.numeric(
+      df$mc_concerned_sideeffects == 1 | 
+      df$mc_concerned_sideeffects == 2
     )
   } else {
-    df$b_hesitant_cov_vaccine <- NA_real_
+    df$b_worried_vaccine_sideeffects <- NA_real_
   }
```

Comment:
Already defined in `variables.R` as `v_worried_vaccine_side_effects`. Should be able to just rename it here.

Reply:
Done.